app/assets/javascripts/containers/MessagesContainer.coffee

```diff
@@ -0,0 +1,9 @@
+mapStateToProps = (state) ->
```

Comment:
Kunnen we niet `passThroughStateToProps` en een `emptyDispatchToProps` maken om dit soort lege functies te beperken?